<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogMeasureElev</class>
 <widget class="QDialog" name="dialogMeasureElev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dialogMeasureElev</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Highlight locations where difference between first-return (a1) DEM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>and last-return (a2) DEM is greater that given threshold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Differences greater than the higher threshold are shown in red</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Differences greater than the lower threshold are shown in yellow</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Toggle highlighting with button on left side of main window</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lower threshold (m)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper threshold (m)</string>
   </property>
  </widget>
  <widget class="QSlider" name="lowerSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="upperSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lowerEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="upperEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lowerSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogMeasureElev</receiver>
   <slot>doLower()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upperSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogMeasureElev</receiver>
   <slot>doUpper()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogMeasureElev</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogMeasureElev</receiver>
   <slot>doApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doLower()</slot>
  <slot>doUpper()</slot>
  <slot>doApply()</slot>
 </slots>
</ui>
